pkgs/development/interpreters/hashlink/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, libpng
+, libjpeg_turbo
+, libvorbis
+, openal
+, SDL2
+, mbedtls
+, pcre
+, libuv
+, libGL
+, libGLU
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hashlink";
+  version = "1.12";
+
+  src = fetchFromGitHub {
+    owner = "HaxeFoundation";
+    repo = "hashlink";
+    rev = version;
+    sha256 = "AiUGhTxz4Pkrks4oE+SAuAQPMuC5T2B6jo3Jd3sNrkQ=";
+  };
+
+  buildInputs = [ libpng libjpeg_turbo libvorbis openal SDL2 mbedtls libuv pcre libGL libGLU sqlite ];
```

Comment:
Suggestion of formatting:
1) list attributes in phases order (fetch => patch => configure => build => check => install)
2) list values in alphabetical order.
3) when a list has more than 3 items, list one item per line.

Reply:
Are attributes not in phase order here?

Reply:
`patches` is patchPhase.
`makeFlags` is configurePhase.